<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="GLWidget" name="myGLWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>250</height>
      </size>
     </property>
     <widget class="QGroupBox" name="groupBoxShape">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>150</width>
        <height>106</height>
       </rect>
      </property>
      <property name="title">
       <string>Shape</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButtonCube">
         <property name="text">
          <string>Cube</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonPyramid">
         <property name="text">
          <string>Pyramid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonSphere">
         <property name="text">
          <string>Sphere</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBoxShape</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>GLWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
